logprep/processor/clusterer/processor.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, name: str, configuration: Processor.Config, logger: Logger):
         self.has_custom_tests = True
 
     def process(self, event: dict):
+        self.matching_rules = []
```

Comment:
it is declared as attribute and set in `__init__`. Am I right if I say, that one of them is redundant?

Reply:
The one in process must exist. The one in __init__ has been kept for readability and better unit tests.